src/widgets/appointments/appointments-detailed-summary.component.tsx

```diff
@@ -109,12 +112,15 @@ export default function AppointmentsDetailedSummary(
         </SummaryCard>
       ) : (
         <EmptyState
-          name={t("Appointments")}
+          name={t("appointments_upper", "Appointments")}
           showComponent={() =>
-            openWorkspaceTab(AppointmentsForm, `${t("Appointments Form")}`)
+            openWorkspaceTab(
+              AppointmentsForm,
+              `${t("appointmentsForm", "Appointments Form")}`
+            )
           }
           addComponent={AppointmentsForm}
-          displayText="This patient has no appointments recorded in the system."
+          displayText={t("appointments_lower", "appointments")}
```

Comment:
upper/lower should be handled with CSS [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)